transform-plugins/src/main/java/co/cask/hydrator/plugin/Normalize.java

```diff
@@ -228,7 +228,7 @@ public NormalizeConfig(String fieldMapping, String fieldNormalizing, String outp
       this.outputSchema = outputSchema;
     }
 
-    void validateConfig() {
+    private void validate() {
       Preconditions.checkArgument(!Strings.isNullOrEmpty(fieldMapping), "Fields to mapped cannot be empty.");
```

Comment:
`fieldMapping` and `fieldNormalizing` are used for schema validation and we planned not to support macros on any properties used for this